dsl/properties/scripts/DockerClient.groovy

```diff
@@ -221,11 +221,46 @@ public class DockerClient extends BaseClient {
         dockerClient.info().content.Swarm.LocalNodeState == "inactive"
     }
 
+    def createNetwork(def serviceDetails){
+
+        String networkName = getNetworkName(serviceDetails)
+        String networkType = getNetworkType(serviceDetails)
+
+        if(!findNetwork(networkName)){
+
+            logger INFO, "Creating $networkName $networkType network."
+
+            def ingress
+
+            if(networkType == "ingress"){
+                ingress = true
+            }else{
+                ingress = false
+            }
+
+            def payload = [
+                    Driver: "overlay",
+                    "IPAM": [
+                            "Driver": "default"
+                    ],
+                    "Ingress" : ingress,
+                    "Scope": "swarm"
```

Comment:
Is the network support restricted to Docker swarm?

Reply:
@urvashisingh Yes, Network exists only in case of Swarm mode.